### PR TITLE
mame & rom-tools 0.224

### PR DESCRIPTION
**Pull request by mbcoguno**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

* Use homebrew's python during building.
* Update license.
* Update comments.

### CODE REVIEW

File: Formula/mame.rb [hunk 1]
```diff
@@ -38,16 +37,18 @@ def install
     # Cut sdl2-config's invalid option.
     inreplace "scripts/src/osd/sdl.lua", "--static", ""
 
-    system "make", "USE_LIBSDL=1",
+    # Use bundled asio and lua instead of latest version.
+    system "make", "PYTHON_EXECUTABLE=#{Formula["python"].opt_bin}/python3",
```
Comment (SMillerDev) [inline Formula/mame.rb:41]:
```suggestion
    system "make", "PYTHON_EXECUTABLE=#{Formula["python@3.8"].opt_bin}/python3",
```

File: Formula/mame.rb [hunk 2]
```diff
@@ -17,13 +17,12 @@ class Mame < Formula
   depends_on "glm" => :build
   depends_on "pkg-config" => :build
   depends_on "pugixml" => :build
+  depends_on "python" => :build
```
Comment (SMillerDev) [inline Formula/mame.rb:20]:
```suggestion
  depends_on "python@3.8" => :build
```

File: Formula/mame.rb [hunk 3]
```diff
@@ -38,16 +37,18 @@ def install
     # Cut sdl2-config's invalid option.
     inreplace "scripts/src/osd/sdl.lua", "--static", ""
 
-    system "make", "USE_LIBSDL=1",
+    # Use bundled asio and lua instead of latest version.
```
Comment (SMillerDev) [inline Formula/mame.rb:40]: If it can use the brewed version it should use that.

File: Formula/rom-tools.rb [hunk 4]
```diff
@@ -30,7 +29,9 @@ def install
     # Cut sdl2-config's invalid option.
     inreplace "scripts/src/osd/sdl.lua", "--static", ""
 
-    system "make", "TOOLS=1",
+    # Use bundled asio instead of latest version.
```
Comment (SMillerDev) [inline Formula/rom-tools.rb:32]: This should use the brewed version

File: Formula/rom-tools.rb [hunk 5]
```diff
@@ -30,7 +29,9 @@ def install
...
+    system "make", "PYTHON_EXECUTABLE=#{Formula["python"].opt_bin}/python3",
```
Comment (SMillerDev) [inline Formula/rom-tools.rb:33]:
```suggestion
    system "make", "PYTHON_EXECUTABLE=#{Formula["python@3.8"].opt_bin}/python3",
```

File: Formula/rom-tools.rb [hunk 6]
```diff
@@ -15,10 +15,9 @@ class RomTools < Formula
   end
 
   depends_on "pkg-config" => :build
+  depends_on "python" => :build
```
Comment (SMillerDev) [inline Formula/rom-tools.rb:18]:
```suggestion
  depends_on "python@3.8" => :build
```

File: Formula/mame.rb (hunk 3 above)
Comment (mbcoguno) [reply Formula/mame.rb:40]: No, it can't.
See #48102 and #57133.

File: Formula/rom-tools.rb (hunk 4 above)
Comment (mbcoguno) [reply Formula/rom-tools.rb:32]: It can't.
See #49695.

Comment (SMillerDev) [reply Formula/rom-tools.rb:32]: Can you link https://github.com/mamedev/mame/issues/5721 then

Comment (mbcoguno) [reply Formula/rom-tools.rb:32]: Then what? Add comments in commit?

Comment (SMillerDev) [reply Formula/rom-tools.rb:32]: Make the comment reference the issue

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**